### PR TITLE
feature: fullfill network scope field in `network ls` and `inspect` 

### PR DESCRIPTION
**Pull request by mathspanda**
Signed-off-by: mathspanda <mathspanda826@gmail.com>

### Ⅰ. Describe what this PR did
Fullfill scope value returned in pouch network list/inpsect.

### Ⅱ. Does this pull request fix one issue?

issue #2501 

### Ⅲ. Why don't you add test cases (unit test/integration test)? (你真的觉得不需要加测试吗？)
Added.

### Ⅳ. Describe how to verify it
``` shell
root@lin-dev-vm2:~# pouch network list
NETWORK ID   NAME     DRIVER   SCOPE
058fce03b8   none     null     local
1bae6b8af5   bridge   bridge   local
d8684bf988   host     host     local
root@lin-dev-vm2:~# pouch network inspect 1bae6b8af5
[
    {
        "Driver": "bridge",
        "IPAM": {
            "Config": [
                {
                    "Gateway": "192.168.5.1",
                    "IPRange": "192.168.5.0/24",
                    "Subnet": "192.168.5.0/24"
                }
            ],
            "Driver": "default"
        },
        "Id": "1bae6b8af52ebd87b4f8a4a7b8b08ec6a096b0acef564f2bf27560ba4baa82da",
        "Name": "bridge",
        "Options": {
            "com.docker.network.bridge.default_bridge": "true",
            "com.docker.network.bridge.enable_icc": "true",
            "com.docker.network.bridge.enable_ip_masquerade": "true",
            "com.docker.network.bridge.host_binding_ipv4": "0.0.0.0",
            "com.docker.network.bridge.name": "p0",
            "com.docker.network.driver.mtu": "1500"
        },
        "Scope": "local"
    }
]
```

### Ⅴ. Special notes for reviews
None

### CODE REVIEW

**Comment by rudyfly** [comment]
UT failed, you can rebase master, and rebuild ci

File: test/cli_network_test.go [hunk 1]
```diff
@@ -48,10 +48,12 @@ func (suite *PouchNetworkSuite) TestNetworkInspectFormat(c *check.C) {
 		c.Errorf("failed to decode inspect output: %v", err)
```
Comment (ZYecho) [inline test/cli_network_test.go]: just confirm, no `network ls` corresponding cli test case added?

Comment (mathspanda) [reply test/cli_network_test.go]: Yeap. I haven't seen any cli test about `network ls`. Maybe needed if `network ls` support some flags.

**Comment by rudyfly** [comment]
LGTM

**Pull request merged**